web/themes/custom/vote_gov/templates/node/node--state-territory--full.html.twig

```diff
@@ -8,46 +8,17 @@
 {% extends 'node.html.twig' %}
 
 {% block content %}
+{% set reg_type = content.field_registration_type | field_value | column('#markup') %}
 
-    {# How to register to vote #}
-    <h2>{{ 'How to register to vote in @state_name' | t({ '@state_name': label | render }) }}</h2>
-    {% if content.field_registration_link | render %}
-      <p>{{ 'Start your online registration on <a href="@link">@state_name\'s election website</a>.' | t({
-          '@link': content.field_override_registration_link | field_value | render | default( content.field_registration_link | field_value | render ),
-          '@state_name': label | render
-        }) }}
-      </p>
-    {% endif %}
+{% if 'Not needed' in reg_type %}
+    {% include '@vote_gov/partial/register/registration-type--not-needed.html.twig' %}
+{% elseif 'Online' in reg_type %}
+    {% include '@vote_gov/partial/register/registration-type--online.html.twig' %}
+{% elseif 'By-mail' in reg_type %}
+    {% include '@vote_gov/partial/register/registration-type--by-mail.html.twig' %}
+{% elseif 'In-person' in reg_type %}
+    {% include '@vote_gov/partial/register/registration-type--in-person.html.twig' %}
+{% endif %}
 
-    {% if content.field_more_info_link | render %}
-      <p>{{ 'You can also register to vote by mail or in person on <a href="@link">@state_name\'s election website</a>.' | t({
-          '@link': content.field_override_more_info_link | field_value | render | default( content.field_more_info_link | field_value | render ),
-          '@state_name': label | render
-        }) }}
-      </p>
-    {% endif %}
-
-    {# Registration deadlines #}
-    <h2>{{ 'Voter registration deadlines in @state_name' | t ({ '@state_name': label | render }) }}</h2>
-    <ul class="registered-resources">
-      {% include '@vote_gov/partial/registration-date-online.html.twig' %}
-      {% include '@vote_gov/partial/registration-date-mail.html.twig' %}
-      {% include '@vote_gov/partial/registration-date-in-person.html.twig' %}
-    </ul>
-
-    {# Check voter registration #}
-    {% if content.field_more_info_link | render %}
-    <h2>{{ 'How to check your voter registration' | t }}</h2>
-      <p>{{ 'You can confirm your voter registration status on <a href="@link">@state_name\'s election website</a>.' | t ({
-        '@link': content.field_override_confirm_reg_link | field_value | render | default( content.field_confirm_registration_link | field_value | render ),
-        '@state_name': label | render
-        }) }}
-      </p>
-    {% endif %}
-    {# NVRF Selector and download button #}
-    <div>
-      {{ drupal_entity('block_content', 3) }}
-    </div>
-
-     <p> {{'Last updated ' | t }}{{ node.revision_timestamp.value | date('F d, Y') }} </p>
+  <p> {{'Last updated ' | t }}{{ node.revision_timestamp.value | date('F d, Y') }} </p>
```

Comment:
trim space and add colon
`<p>{{'Last updated:' | t }}{{ node.revision_timestamp.value | date('F d, Y') }}</p>`